cmd/flagsaver/flagsaver.go

```diff
@@ -0,0 +1,36 @@
+// Package flagsaver provides a simple way to save and restore flag values.
+// TODO(RJPercival): Move this to its own GitHub project.
+//
+// Example:
+//   func TestFoo(t *testing.T) {
+//     defer flagsaver.Save().Restore()
+//     // Test code that changes flags
+//   } // flags are reset to their original values here.
+package flagsaver
+
+import "flag"
+
+// Stash holds flag values so that they can be restored at the end of a test.
+type Stash struct {
```

Comment:
It works fine as a struct, but since you don't have other fields an alias could have worked too. It's fine as it is.